doc/draft-rquattle-spinel-unified.txt

```diff
@@ -2878,9 +2878,9 @@ Quattlebaum & woodyatt  Expires November 2, 2017               [Page 51]
                | Fields:  | Length | CoAP | locator | port |
                +----------+--------+------+---------+------+
 
-   This property allows the host to send and receive border-agent-
+   This property allows the host to send and receive border-router-
    related CoAP requests/responses from the NCP's RLOC address.  This
-   allows the host driver to implement a Thread border agent.
+   allows the host driver to implement a Thread border router.
```

Comment:
how about: "This property allows the host to send and receive TMF messages from the NCP's RLOC address and support Thread-specific border router functions."